.github/workflows/daily-jobs.yml

```
@@ -0,0 +1,43 @@
+name: Daily container builds for unstable distro releases
+
+on:
+  # These push/pr options will be removed one they are proven to work (one PR/push ought to do it)
+  pull_request:
+    branches: main
+  push:
+    branches: main
```

These two conditions are here to get the workflow to trigger now, and show that it works. The can be removed prior to merging, by incorporating this suggestion
```suggestion
-  # These push/pr options will be removed one they are proven to work (one PR/push ought to do it)
-  pull_request:
-    branches: main
-  push:
-    branches: main
```